### DISCUSSION TITLE
IOTA stolen this morning from my app, IOTA LOttery, is it a scam or true!

### DISCUSSION THREAD

**Discussion by donnalife** [iotaledger/firefly#8062] [Help & Support]
This morning I was very stupid, I opened the firefly, to update and else, and appear that I had won 2000 IOTA lottery, I did all that required.
Now I go into the platform,  and I only have 0,23 usd, from 5500 usd plus, or almost 20,000 IOTAS stolen!!!
Please help

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
In your wallet go to:
1) Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
3) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/9e9f7056-e167-41c8-b00b-cba65a3bc87c)

**Reply by donnalife** [reply]
I lost my NFT, but the question, how this appear in my Firefly app =? why IOTA fundation its letting this happen, you open you App, and you think its safe, but it isnt, how you know its not IOTa fundation, please help

**Reply by pablo-hp** [reply]
IOTA IS SCAM. SIMPLE. 
I wonder how, without me sending my wallet address to anyone, someone can magically discover my wallet address and send me funds.
PRIVACY - ZERO!

**Reply by begonaalvarezd** [reply]
I am very sorry you are a victim of the iota lottery scam. We cant prevent people from gifting you an NFT, addresses are public so anybody can pick an address and make a gift, unfortunately in this case is an NFT with a scam message. The whole point of self custody wallets in crypto is that you are you own bank so you have to take good care of your keys because you are the only one holding them. The ecosystem is full of warnings like "do not share your keys with anybody, do not enter your keys in any random apps, ..." At this point we can only suggest you to report it to your local authorities.
